public/settings.json

```diff
@@ -3,5 +3,6 @@
   "google": {
     "clientId" : "147813579094-8qbv6canv6167qlga4k9788dg0qmgokn.apps.googleusercontent.com",
     "cookiePolicy": "single_host_origin"
-  }
+  },
+  "url": "https://hermes.api.dev.k8s.nerds.sh/graphql"
```

Comment:
What does this `url` refer to? The url of the client? The url of the api? A storage bucket? 

Let's clarify the naming here.

Reply:
rename to api and move.

Reply:
done